attestation-service/attestation-service/src/policy_engine/mod.rs

```diff
@@ -53,4 +61,8 @@ pub trait PolicyEngine {
     ) -> Result<HashMap<String, (PolicyDigest, EvaluationResult)>>;
 
     async fn set_policy(&mut self, input: SetPolicyInput) -> Result<()>;
```

Comment:
i know this isn't part of your PR, but I wonder if we should rename policy to `add_policy` since we support multiple policies.

Reply:
I think using set policy here is more accurate because we not only use this API to add new policies, but also to update some already set policies.